src/http/client/http.ts

```diff
@@ -18,31 +18,31 @@ export class HttpClient {
     this.authToken = config.authToken
   }
 
-  public async get(req: IHttpGETRequest): Promise<IHttpResponse> {
+  public async get(req: IHttpGETRequest, headers?: object): Promise<IHttpResponse> {
```

Comment:
type object ? If using `àny` or `object`, you can deduce that something is wrong with the typing.

Reply:
Actually I have been checking Axios impl to copy its headers type and I found this:
```
export interface AxiosRequestConfig {
  url?: string;
  method?: Method;
  baseURL?: string;
  transformRequest?: AxiosTransformer | AxiosTransformer[];
  transformResponse?: AxiosTransformer | AxiosTransformer[];
  headers?: any;
```

```